src/Makefile

```diff
@@ -694,6 +706,11 @@ rshapes.o : rshapes.c raylib.h rlgl.h
 rtextures.o : rtextures.c raylib.h rlgl.h utils.h
 	$(CC) -c $< $(CFLAGS) $(INCLUDE_PATHS)
 
+RGFW.o: 
```

Comment:
Is it possible to avoid compiling this unit? One of the benefits of RGFW would be compiling it directly into `rcore_desktop_rgfw.c`.

Reply:
I don't know if that's possible due to naming conflicts with winapi.